packages/manager/fragments/sidebar/src/shims-tsx.d.ts

```diff
@@ -0,0 +1,13 @@
+import Vue, { VNode } from 'vue';
```

Comment:
Out of curiosity  here, what's is the purpose of both files:
- packages/manager/fragments/sidebar/src/shims-tsx.d.ts
- packages/manager/fragments/sidebar/src/shims-vue.d.ts

Is it supposed to be used elsewhere?

Reply:
Both files has been auto generated by `vue-cli`, I think there are needed for Typescript/Vue so TSX and Vue files have correctly inferred types

Reply:
Yes exactly, that's idea. 

TS doesn't understand Vue files that are usually handled by a vue-loader, if you don't add shims, when importing a vue/tsx module you'll likely encouter an error like "Cannot find module", the shims allows your IDE to understand what a .vue/.tsx file is. 

It's mainly IDE oriented, so depending or your IDE configuration (extensions etc) it may not block the app from running. Though you won't benefit from ts linter if these files are missing. 

Usually we also need to tweak these files when testing with vue-test-utils as the test API doesn't always have the same definitions as the Vue API.